### PR TITLE
DevTest: Add item meta editor

### PR DESCRIPTION
**Pull request by Wuzzy2**
Adds an Item Meta Editor to DevTest.

This is a tool which allows you to edit the metadata of items. It's like the Node Meta Editor, but for items.

## How to test

Get the Item Meta Editor and put an item in the next slot. Punch or place with the Item Meta Editor and an editor window appears. Here you can get and set metadata.

The Item Meta Editor works like the Node Meta Editor.

Try out setting `color`, `palette_index`, `description`, etc.

### CODE REVIEW

File: games/devtest/mods/testtools/init.lua [hunk 1]
```diff
@@ -750,11 +761,47 @@ minetest.register_tool("testtools:node_meta_editor", {
 		node_meta_posses[user:get_player_name()] = pos
 		local meta = minetest.get_meta(pos)
 		local inv = meta:get_inventory()
-		show_node_meta_formspec(user, pos, "", "", get_node_meta_keylist(meta, user:get_player_name(), true))
+		show_meta_formspec(user, "node", pos, "", "", get_meta_keylist(meta, user:get_player_name(), true))
 		return itemstack
 	end,
 })
 
+local get_item_next_to_wielded_item = function(player)
```
Comment (appgurueu) [inline games/devtest/mods/testtools/init.lua:769]: Why don't you use the `local function` syntactic sugar consistently?

**Comment by Wuzzy2** [comment]
Done, appgurueu.

**Comment by SmallJoker** [comment]
Suggestion: Replace or remove the "Get Value" button with "Delete". The list selections already perform an update. If it is supposed to update, the button could as well be named "Refresh".

Other than that - the changes work as expected and the code looks good.

**Comment by Wuzzy2** [comment]
OK, the "Get Value" button is gone.

**Review by SmallJoker — APPROVED** [review]

**Review by rubenwardy — APPROVED** [review]

**Pull request merged**